src/components/AuthLayout.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Form, NavLink } from 'react-router-dom';
+import PropTypes, { bool } from 'prop-types';
+
+import Register from './Register';
+import Login from './Login';
+
+import './auth-layout.css';
+
+const AuthLayout = (props) => {
+    const isRegistration = props.isRegistration;
+
+    return (
+        <>
+            <h1>{props.name}</h1>
+            <section id="auth-layout">
+                <form>
+                    {isRegistration ? <Register/> : <Login/>}
+                    <button className="btn btn-primary">{props.name}</button>
+                </form>
+                <div>
+                    <span>{isRegistration ? 'already' : 'not'} a member ?&nbsp;</span>
```

Comment:
This isn't optimal. I don't the creation of this AuthLayout component is worth it, I'm envisioning something more like this in terms of re-usability:

```
const Login = () => (
    <AuthLayout>
        <Form>
             <Input/>
             <Input/> 
        </Form>
    </AuthLayout>
);
```

This promotes super isolated - and therefore - highly reusable components

Reply:
Please address this comment first. Then I'll review again, as quite a lot is going to change

Reply:
alright will look into that